Stanislav_Sadkov/lesson_2_hw/app.js

```diff
@@ -0,0 +1,205 @@
+const fs = require('fs');
+const filename = require('minimist')(process.argv.slice(2)).file;
+const readline = require('readline');
+const reader = readline.createInterface(process.stdin, process.stdout);
+
+const app = {
+    //Создаем массив с картами
+    getCardsArr : ()=>{
+        return new Map([
+            [1, 1], [2, 2], [3, 3], [4, 4],
+            [5, 5], [6, 6], [7, 7], [8, 8], [9, 9],
+            [10,10], ['Королева', 10], ['Король', 10], ['Валет', 10],
+            ['Туз', 11]
+            ]);
+        },
+    //Создаем массив с мастями
+    suits : new Map([
+            ['diamonds', '♦'],
+            ['hearts', '♥'],
+            ['spades', '♠'],
+            ['clubs', '♣']
+        ]),
+    //Генерация полной колоды
+    generateDeck : function (){
+            let fullDeck =  new Map();
+            for(let suit of this.suits.keys()) {
+                fullDeck.set(suit, this.getCardsArr());
+            }
+            return fullDeck;
+    },
+    getRandomCard : function(){
+        //Отдаем рандомную карту и удаляем из колоды
+        let deck = this.deck;
+        let suits = Array.from(deck);
+        let suit = suits[Math.floor(Math.random() * suits.length)];
+        let cards = Array.from(suit[1]);
+        let suitKey = suit[0];
+        let cardRandom = cards[Math.floor(Math.random() * cards.length)];
+        let cardRandomKey = cardRandom[0];
+        deck.get(suitKey).delete(cardRandomKey);
+        return cardRandom;
+    },
+    player : {
+        points: 0,
+        wins: 0
+    },
+    ai : {
+        points: 0,
+        wins: 0
+    },
+    plays: 0,
+    deck: null,
+    play: function(){
+        const that = this;
+        reader.question('\nХотите начать игру? (да/нет) ', function(text) {
+            if (text === "нет") {
+                console.log('Игра не начата');
+                reader.close();
+            }else if(text === "да"){
```

Comment:
Проверьте везде - стоит привести все к одному виду кавычек
```suggestion
            }else if(text === 'да'){
```